### PR TITLE
Collect vpp postmortem data (simplified)  #738

### PR DESCRIPTION
**Pull request by stepan-anokhin**
Signed-off-by: Stepan Anokhin <stepan.anokhin@gmail.com>

## Description
Collect vpp postmortem data. 

## Motivation and Context
See #738

## How Has This Been Tested?
- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by stepan-anokhin** [comment]
@edwarnicke please review. 

This is a simplified alternative of #762.

File: .circleci/config.yml [hunk 1]
```diff
@@ -132,10 +132,19 @@ jobs:
           environment:
             KUBECONFIG: /home/circleci/project/data/kubeconfig
             GO111MODULE: "on"
+      - run:
+          when: always
```
Comment (nickolaev) [inline .circleci/config.yml]: Should this be `when: on_dail` instead?

File: dataplane/vppagent/scripts/vpp_listener.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
```
Comment (nickolaev) [inline dataplane/vppagent/scripts/vpp_listener.py]: Why not Go?

**Comment by nickolaev** [comment]
@stepan-anokhin is this obsoleting #762?

File: dataplane/vppagent/scripts/vpp_listener.py (hunk 2 above)
Comment (stepan-anokhin) [reply dataplane/vppagent/scripts/vpp_listener.py]: It is derived from the original [supervisord_kill.py](https://github.com/ligato/vpp-agent/blob/master/docker/prod/supervisord_kill.py). I don't see a need to rewrite it in go here.

File: .circleci/config.yml (hunk 1 above)
Comment (stepan-anokhin) [reply .circleci/config.yml]: I theory vpp crash may not cause test failure. 
Either way this is a very cheap operation.

**Comment by stepan-anokhin** [comment]
@nickolaev yep, this one obsoletes #762

**Pull request merged**